src/common/recordbatch/src/lib.rs

```diff
@@ -246,7 +246,7 @@ impl<S> RecordBatchStreamWrapper<S> {
     }
 }
 
-impl<S: Stream<Item = Result<RecordBatch>> + Unpin> RecordBatchStream
+impl<S: Stream<Item = Result<RecordBatch>> + Send + Unpin> RecordBatchStream
```

Comment:
Why do we need additional `Send` bound?

Reply:
I thought we need the send to satisfy the Send trait bound for SendableRecordBatchStream but it turns out we don't need it. I will revert the changes src/common/recordbatch/src/lib.rs
`pub type SendableRecordBatchStream = Pin<Box<dyn RecordBatchStream + Send>>;`